.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,23 @@
 <!-- Provide a general summary of your changes in the Title above -->
 
+
 ## Description
-<!-- Describe your changes in detail -->
+<!-- Describe your changes in detail. -->
+<!-- Please indicate if this change breaks or may break existing functionality, so we can mark it as 'breaking'. -->
+
 
 ## Motivation and Context
 <!-- Why is this change required? What problem does it solve? -->
 <!-- If it fixes an open issue, please link to the issue here. -->
 
-## Types of Changes
-<!-- Please select all items that apply either now or after creating the pull request: -->
-- [ ] Documentation update
-- [ ] Bug fix
-- [ ] New feature
-- [ ] Breaking change<sup>1</sup>
-
-<sup>1</sup>The change breaks (or has the potential to break) existing functionality.
 
 ## Checklist:
-<!-- Please select all items that apply either now or after creating the pull request. -->
+<!-- Please complete the checklist before the PR can be merged. Reviews can begin while the checklist is in progress. -->
 <!-- If you are unsure about any of these items, do not hesitate to ask! -->
 - [ ] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md).
 - [ ] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac/blob/master/ContributorAgreement.md).
 - [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
 - [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
 - [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
-
-If necessary:
-- [ ] I have updated the API documentation as part of the package doc-strings.
-- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
-- [ ] I have updated the [changelog](https://github.com/glotzerlab/signac/blob/master/changelog.txt) and added all related issue and pull request numbers for future reference (if applicable). See example below.
-
-<!-- Example for a changelog entry: `Fix issue with launching rockets to the moon (#101, #212).` -->
+- [ ] If this PR changes user-facing features or breaks existing APIs, I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and included a link here.
```

Comment:
Still not sure whether it is clear that this should be checked in case this does _not_ apply. Should the reviewers do that?

Reply:
API breaks don't always require a pull request to signac-docs. I'd even bet that _most_ API breaks do not require changes in the framework documentation.

I agree with Simon that it's not clear that the intended action for most pull requests is to check this box without needing to make a separate change in the documentation. I am having trouble coming up with a wording that is concise and indicates that this box should be checked by default.

Given the difficulty in describing this concisely and the relatively small number of PRs to which it pertains, should we remove the signac-docs item from the checklist? Changes in signac-docs happen pretty infrequently, and perhaps it's better to remove this checkbox and leave it to committers to request that as a part of review. This could be described in the Contributing Guidelines instead.

Reply:
I agree it's still not clear.
I think we should keep SOME reminder about the docs because failing to update docs (and still passing review) is what precipitated this PR. Even though they are small in number, they are more important to get right.

Maybe create some visible text or subsection called "Do you think the documentation needs to be updated?"

Reply:
We’re probably just as guilty of forgetting the package docs, let alone the framework docs. Maybe this:

```suggestion
- [ ] The package documentation and [framework documentation](https://docs.signac.io/) in [signac-docs](https://github.com/glotzerlab/signac-docs) are up to date with these changes.
```

Reply:
I like this a lot!